### ISSUE TITLE
Upstream: a11y(critical) studio course outline

### ISSUE THREAD

**Issue by bmtcril** [openedx/end-to-end-tests#50]
2 critical violations:

https://github.com/openedx/frontend-app-authoring/issues/3231
